docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml

```diff
@@ -277,6 +271,59 @@ ECLCC_PATH
               </tgroup>
             </informaltable></para>
         </sect3>
+
+        <sect3 id="CT-CLI-GIT-Support">
+          <title>Git Support</title>
+
+          <para>Starting with the HPCC Systems version 8.4, the
+          <emphasis>--main</emphasis> option has been extended to support
+          compiling a query directly from a Git repository. When invoked, It
+          retrieves the ECL code specified from the Git repository, compiles
+          the code, and runs the query. Only the checkout will be done on the
+          remote ECLCCServer rather than on the client machine. Here is the
+          syntax to reference a repository location:</para>
+
+          <programlisting>&lt;protocol:&gt;//&lt;urn&gt;/&lt;user&gt;/&lt;repository&gt;#version </programlisting>
+
+          <para>While the <emphasis>#version</emphasis> component specifies
```

Comment:
This sentence doesn't read clearly.  Possibly delete "specifies and" and replace "could" with "can"